### PR TITLE
PP-7854 Make Card Payment Smoke Test

### PR DESCRIPTION
**Pull request by rorymalcolm**
- Add the first smoke test, a runner for it, and the stubs to stub out calls to Synthetics

### CODE REVIEW

File: run-local/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+var makeCardPayment =  require("../make-card-payment");
```
Comment (sfount) [inline run-local/index.js]:
```suggestion
const makeCardPayment =  require("../make-card-payment");
```

File: run-local/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+let makeCardPayment =  require("../make-card-payment");
+
+process.env.LOCAL_SMOKE_TEST = "true"
```
Comment (sfount) [inline run-local/index.js:3]: 👍  would recommend the local start script sets it up to use polyfills as apposed to needing environment + the right file to be run, this looks like it does it

File: stubs/syntheticsLoggerStub/index.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+exports.info = function (log) {
+    console.log(log)
+}
```
Comment (sfount) [inline stubs/syntheticsLoggerStub/index.js:3]:
```suggestion
exports.info = console.log
```

File: stubs/syntheticsLoggerStub/package.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "syntheticsloggerstub",
```
Comment (sfount) [inline stubs/syntheticsLoggerStub/package.json:2]: I'm guessing this is an artefact of including it as a dev dependency? does it need to stick around with the direct ref?

File: stubs/syntheticsStub/index.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
+let browser 
+let page
+
+const puppeteer = require("puppeteer")
+
+exports.getPage = async () => {
+    browser = await puppeteer.launch({
```
Comment (sfount) [inline stubs/syntheticsStub/index.js:7]:
```suggestion
    let page
    const browser = await puppeteer.launch({
```

File: stubs/syntheticsStub/package.json [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{
```
Comment (sfount) [inline stubs/syntheticsStub/package.json:1]: Similar question about the use of this

**Review by sfount — COMMENTED** [review]
If your team feel like it continues to be a reasonable idea I'd suggest throwing this through `standard` linter + fix to keep code consistent and not bother coming up rules for each smaller repo

**Review by sfount — APPROVED** [review]
Changes look ace 👍  Final thought is that it might be useful to have one or two comment lines (👀 ) that explain why we have local wrapper files and how that behaviour differs in the expected runtime environment (magic lambda land). That avoids commenting what the code is doing but gives some helpful context for future us.

**Comment by rorymalcolm** [comment]
> Changes look ace 👍  Final thought is that it might be useful to have one or two comment lines (👀 ) that explain why we have local wrapper files and how that behaviour differs in the expected runtime environment (magic lambda land). That avoids commenting what the code is doing but gives some helpful context for future us.

Going to expand the README with info on this alongside comments, will do that in a new PR

**Pull request merged**